types/express-ua-middleware/index.d.ts

```diff
@@ -3,7 +3,7 @@ import { RequestHandler } from "express-serve-static-core";
 /**
  * Express middleware to parse user-agent header
  */
-declare function ua(): RequestHandler;
```

Comment:
wrong type — it's actually RequestHandler, not a function returning it
see https://github.com/davidsdevel/express-ua-middleware/blob/master/index.js